requirements-common.txt

```diff
@@ -53,3 +53,8 @@ uWSGI==2.0.17.1
 validate-email==1.3
 webencodings==0.5.1
 Whoosh==2.7.4
+django-reversion==3.0.4
+django-reversion-compare==0.8.6
+django-cors-headers==3.0.2
+djangorestframework-jwt==1.11.0
+django-rest-swagger==2.2.0
```

Comment:
Please add these pakages to requirements-py3.txt file.

Reply:
ok